scripts/release-redhat-certified.sh

```diff
@@ -2,6 +2,18 @@
 
 set -eou pipefail
 
+echo -n "Determining SHA for arm64 ... "
+IMG_SHA_ARM64=$(docker \
+  manifest inspect quay.io/mongodb/mongodb-atlas-kubernetes-operator:${VERSION}-certified | \
+  jq --raw-output '.manifests[] | select(.platform.architecture == "arm64") | .digest')
+echo ${IMG_SHA_ARM64}
```

Comment:
```suggestion
echo "SHA ARM64 ${IMG_SHA_ARM64}"
```

Reply:
@igor-karpukhin it's in the same line as the `Determining` stanza, so the output is:
```
Determining SHA for arm64 ... sha256:cee0e5c5c4aa4957d0526e0c9d739067dca737e6de776b95ce44edf7dfe9a6b4
Determining SHA for amd64 ... sha256:af8713ce9ccf38800b67ab7ae2bf2ca6d904be8bf7714e33c540a06392fe3e0d
```